bpf/kmesh/workload/include/endpoint.h

```diff
@@ -29,7 +29,13 @@ static inline int endpoint_manager(
     ret = backend_manager(kmesh_ctx, backend_v, service_id, service_v);
     if (ret != 0) {
         if (ret != -ENOENT)
-            BPF_LOG(ERR, ENDPOINT, "backend_manager failed, ret:%d\n", ret);
+            BPF_LOG(
+                ERR,
+                ENDPOINT,
+                "backend_manager failed, svc %u backend %u ret:%d\n",
```

Comment:
What does the value of `ret` mean? 
Is it possible to delete it?

Reply:
we need to print the return value to help debug